.github/home.css

```
@@ -249,4 +257,246 @@ footer {
   font-size: 16px;
   font-weight: 400;
   text-decoration: none;
+}
+
+.mobile {
+  display: none;
+}
+
+@media screen and (min-width: 768px) and (max-width: 1199px) {
+  * {
+    max-width: 1199px;
+  }
+
+  body {
+    max-width: 1199px;
+  }
+
+  nav {
+    padding: 20px 32px;
+    margin: 0;
+  }
+
+  .nav {
+    margin: 0;
+    padding: 0;
+  }
+
+  .nav a {
+    margin: 0;
+  }
+
+  .nav > a > img {
+    width: 133px;
+    height: 24px;
+  }
+
+  .login_bt {
+    width: 128px;
+    padding: 16px 20px;
+    gap: 10px;
+  }
+
+  header {
+    display: flex;
+    width: 1199px;
+    padding: 100px 250px 0;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 40px;
+  }
+
+  header > h1 {
+    height: fit-content;
+    margin: 0;
+    font-size: 56px;
+    text-align: center;
+  }
+
+  header > a {
+    display: flex;
+    width: 350px;
+    padding: 16px 20px;
+    justify-content: center;
+    align-items: center;
+    gap: 10px;
+  }
+
+  .header_img {
+    width: 698px;
+    height: 390px;
+    justify-content: center;
+  }
+  
+  .header_img > img {
+    width: 698px;
+  }
+
+  section {
+    height: 440px;
+    padding: 80px 250px 50px;
+  }
+  
+  .section_contents {
+    width: 698px;
+    height: 315px;
+    gap: 51px;
+  }
+  
+  .section_text {
+    width: 262px;
+  }
+  
+  
+  .section_text>h2 {
+    font-size: 2.5rem;
+  }
+  
+  .section_text>p {
+    font-size: 13.5px;
+  }
+  
+  .section_contents > div > img {
+    width: 385px;
+  }
+  
+  .footer_contents {
+    width: 991px;
+  }
+}
+
+@media screen and (min-width: 375px) and (max-width: 767px) {
+  * {
+    max-width: 767px;
+  }
+
+  nav {
+    padding: 13px 32px;
+  }
+
+  .nav {
+    padding: 0;
+  }
+
+  .login_bt {
+    width: 80px;
+    height: 20px;
+    margin: 0;
+    font-size: 12px;
+  }
+
+  .nav > a > img {
+    width: 89px;
+    margin: 0;
+  }
+
+  .nav a {
+    margin: 0;
+  }
+
+  header {
+    width: 767px;
+    padding: 28px 32px 0px;
+    gap: 24px;
+  }
+
+  header > h1 {
+    height: fit-content;
+    margin: 100px 0 0 0;
```

```suggestion
    margin-top: 100px;
```
특정 영역에 margin을 주고 싶다면 이 방법이 더 효율적입니다